app/grandchallenge/notifications/static/js/notifications/bulk_delete_update.js

```diff
@@ -1,6 +1,8 @@
-window.toggleCheckboxes = function toggleCheckboxes(source) {
+document.getElementById("SelectCheckboxes").addEventListener("click", toggleCheckboxes);
+
+function toggleCheckboxes(source) {
     $('input[name="checkbox"]').each(function () {
-        this.checked = source.checked;
+        this.checked = source.srcElement.checked;

```

Comment:
I think that `srcElement` is deprecated for `target` https://developer.mozilla.org/en-US/docs/Web/API/Event/srcElement